src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,221 @@
+import * as React from 'react'
+import getValidProps from '@helpscout/react-utils/dist/getValidProps'
+import propConnect from '../PropProvider/propConnect'
+import { classNames } from '../../utilities/classNames'
+import { namespaceComponent } from '../../utilities/component'
+import { noop } from '../../utilities/other'
+import { COMPONENT_KEY } from './Pagination.utils'
+import pluralize from '../../utilities/pluralize'
+
+import {
+  PaginationUI,
+  InformationUI,
+  NavigationUI,
+  RangeUI,
+  ButtonIconUI,
+} from './Pagination.css.js'
+import Text from '../Text'
+import Icon from '../Icon'
+import Button from '../Button'
+
+export interface Props {
+  activePage: number
+  className?: string
+  innerRef: (node: HTMLElement) => void
+  onChange: (nextPageNumber: number) => void
+  rangePerPage: number
+  separator?: string
+  showNavigation?: boolean
+  pluralizeSubject?: string
+  subject: string
+  totalItems: number
+}
+
+export class Pagination extends React.PureComponent<Props> {
```

Comment:
This module was very well worded, all the function names make total sense. 👍

Reply:
thank you 😄